core/modes/language_modes.py

```diff
@@ -122,3 +139,8 @@ def code_clear_language_mode():
         global forced_language
         forced_language = ""
         ctx.tags = []
+
+    def code_show_forced_language_mode():
+        """Show the active language for this context"""
+        if forced_language
```

Comment:
Should this have in else showing a notification that there is no forced language?